### PR TITLE
chore(ci): update current year

### PR DESCRIPTION
**Pull request by sc979**
## Description

Update current year

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [x] 24.04.x (master)

### CODE REVIEW

File: src/PhpCsFixer/PhpCsFixerLicense.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 class PhpCsFixerLicense
 {
-    private const YEAR = 2023;
+    private const YEAR = 2024;
```
Comment (callapa) [inline src/PhpCsFixer/PhpCsFixerLicense.php:28]: No need to change for next year.
```suggestion
    private const YEAR = (new DateTime())->format('Y');
```

**Review by a-launois — APPROVED** [review]

File: src/PhpCsFixer/PhpCsFixerLicense.php (hunk 1 above)
Comment (sc979) [inline src/PhpCsFixer/PhpCsFixerLicense.php:28]:
```suggestion
    private const YEAR = (new DateTime())->format('Y');
```

[the remark this answers is not in the document]
Comment (sc979) [reply src/PhpCsFixer/PhpCsFixerLicense.php:28]: removing this constant will break php-cs-fixer scripts

File: src/PhpCsFixer/PhpCsFixerLicense.php [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
 class PhpCsFixerLicense
 {
-    private const YEAR = 2023;
+    private const YEAR = (new DateTime())->format('Y');
```
Comment (sc979) [inline src/PhpCsFixer/PhpCsFixerLicense.php:28]:
```suggestion
    private const YEAR = 2024;
```

**Review by callapa — APPROVED** [review]

**Pull request merged**